kurtosis/src/nodes/execution/execution.star

```diff
@@ -107,10 +121,11 @@ def add_bootnodes(node_module, config, bootnodes):
 
     return config
 
-def deploy_nodes(plan, configs):
+def deploy_nodes(plan, configs, is_full_node = False):
     service_configs = {}
     for config in configs:
         service_configs[config["name"]] = service_config_lib.create_from_config(config)
+        service_configs[config["name"]] = service_config_lib.create_from_config(config, is_full_node)
```

Comment:
This is called twice, even when it's unnecessary

Reply:
Thanks for pointing out, addressed it. I may have messed this while resolving merge conflicts.